src/context/App.tsx

```diff
@@ -244,6 +247,9 @@ export const AppProvider = ({ children }: { children: ReactNode }) => {
     if (existing.settings) {
       setKeyboardShortcut(existing.settings.keyboardShortcut);
       setSettings({ ...defaultSettings, ...existing.settings });
+      webFrame.setZoomLevel(
+        zoomPercentageToLevel(existing.settings.zoomPercentage),
+      );
```

Comment:
@setchy the error is because of these lines here. What I am thinking is doing something like this, but not entirely sure. Any suggestions?
```suggestion
      webFrame.setZoomLevel(
        zoomPercentageToLevel(existing.settings.zoomPercentage) || 0,
      );
```

Reply:
Ah... so it's zooming too much on startup

Reply:
Fixed it! My thought process was correct at the time, but what I wrote differed from what I thought 😅

Reply:
Nice! - the startup issue is fixed!